KsApi/models/GraphBacking.swift

```diff
@@ -21,3 +22,20 @@ public struct GraphBacking: Swift.Decodable, Equatable {
     case preauth
   }
 }
+
+extension GraphBacking.Project {
```

Comment:
I don't think you need this extension, `Decodable` should be able to infer the keys and the values.

We should however update `GraphBackingTests` to include the property that was added.